src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
```

Comment:
Consider using a variable or constant for the border color instead of hardcoding it. This will make it easier to change the color in the future. 🎨